src/routes/api/users/[userId]/demos/index.ts

```diff
@@ -4,16 +4,15 @@ import config from '../../../../../../mikro-orm.config';
 import { LoadStrategy, MikroORM } from '@mikro-orm/core';
 import { Expose, plainToInstance } from 'class-transformer';
 import { validateSync } from 'class-validator';
-import { classTransformOptions, classValidationOptions } from '../../../../.utils';
```

Comment:
What is the rational behind getting rid of `classTransformOptions` and `classValidationOptions`?

Reply:
This is potentially dangerous and can expose unintended fields from upstream API calls (eg. Weather) that we proxy.